assets/react/v3/entries/course-builder/components/curriculum/QuestionConditions.tsx

```diff
@@ -104,12 +104,24 @@ const QuestionConditions = () => {
                 <FormSwitch
                   {...controllerProps}
                   label={__('Multiple Correct Answer', 'tutor')}
-                  onChange={() => {
+                  onChange={(value) => {
                     calculateQuizDataStatus(activeDataStatus, QuizDataStatus.UPDATE) &&
                       form.setValue(
                         `questions.${activeQuestionIndex}._data_status`,
                         calculateQuizDataStatus(activeDataStatus, QuizDataStatus.UPDATE) as QuizDataStatus,
                       );
+
+                    if (!value) {
```

Comment:
Add a comment for future reference here. Explaining why we need this setValue.

Reply:
Done